cmd/keyaliases/crtka.go

```diff
@@ -58,7 +58,7 @@ var CreateCmd = &cobra.Command{
 				ignoreExpiry,
 				ignoreNewLine,
 				selfFile)
-		case "pem":
+		case "keycertfile":
```

Comment:
Wouldnt this be a breaking change for customers using "pem"

Should we support both pem and keycertfile?

Reply:
Yes it would be. I suppose we can keep pem for backward compatibility.

Reply:
Done